Additional documentation about Open Distro for Elasticsearch Kibana reports feature (#2266)
epiphany-platform/epiphany: docs/home/howto/LOGGING.md

```
After applying this policy every new index created under this one will apply to it. There is also possibility to apply policy to already existing policies by assigning them to policy in Index Management Kibana panel.

> ## How to export Kibana reports to csv format

Reviewer: ```suggestion
## Export Kibana reports to CSV format
```

Reviewer 2: I would leave 'How to' since it's 'how to' section and we have structure like this here: https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/HOWTO.md

> Since v0.10 Epiphany provide posibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for Elasticsearch Kibana reports feature.

Reviewer: ```suggestion
Since v0.10 Epiphany provides possibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for Elasticsearch Kibana reports feature.
```

More details about the plugin and how to export reports:  

> https://opendistro.github.io/for-elasticsearch-docs/docs/kibana/reporting

Reviewer: hyperlink?

Author: > Could you please make lines max 120 chars?

Sure, good point, I was following the standard from Elasticsearch section which is wrong so I also improved a little bit the section with elasticsearch.

> Note: Currently in Open Distro for Elasticsearch Kibana the following plugins are installed and enabled by default: security, alerting, anomaly detection, index management, query workbench, notebooks, reports, alerting, gantt chart plugins.

Reviewer: Check how other 'notes' are formatted in this document

> You can easly check enabled default plugins for Kibana using the following command on the logging machine:

Reviewer: ```suggestion
You can easily check enabled default plugins for Kibana using the following command on the logging machine:
```

After applying this policy, every new index created under this one will apply to it. There is also possibility to apply
policy to already existing policies by assigning them to policy in Index Management Kibana panel.

> ## How to Kibana reports to CSV format

Reviewer: ```suggestion
## How to export Kibana reports to CSV format
```

> Since v0.10 Epiphany provides possibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for

Reviewer 2: ```suggestion
Since v1.0 Epiphany provides the possibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for
```

`/usr/share/logstash/vendor/bundle/jruby/2.5.0/gems/logstash-input-elasticsearch-*/lib/logstash/inputs/elasticsearch.rb`

Check more details about configuration of input plugin in the
[documentation](https://www.elastic.co/guide/en/logstash/current/plugins-inputs-elasticsearch.html)

Check more details about configuration of output plugin in the
[documentation](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-csv.html)

`Note: Currently input plugin doesn't officialy support skipping certificate validation for secure connection to Elasticsearch.`

> 

Reviewer: This part is located above, lines 258-266
